src/Normalizer/JeMengageWebSurveyDenormalizer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Normalizer;
+
+use App\Entity\Jecoute\LocalSurvey;
+use App\Entity\Jecoute\NationalSurvey;
+use App\Entity\Jecoute\Survey;
+use App\Jecoute\SurveyTypeEnum;
+use http\Exception\UnexpectedValueException;
+use Symfony\Component\Serializer\Normalizer\AbstractNormalizer;
+use Symfony\Component\Serializer\Normalizer\DenormalizerAwareInterface;
+use Symfony\Component\Serializer\Normalizer\DenormalizerAwareTrait;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+class JeMengageWebSurveyDenormalizer implements DenormalizerInterface, DenormalizerAwareInterface
```

Comment:
There is no other survey denormalizer and it make just a base transformation by type, it could be named `SurveyDenormalizer`:
```suggestion
class SurveyDenormalizer implements DenormalizerInterface, DenormalizerAwareInterface
```